d3-build (#19643)
supabase/supabase: apps/www/_blog/2023-12-13-postgrest-12.mdx

---

> title: "What's new in PostgREST 12?"

Reviewer: ```suggestion
title: "PostgREST 12"
```

> description: 'PostgREST 12 is out and we take a look at some of our favorite new features like JWT Caching and Aggregate Functions'

Reviewer: ```suggestion
description: 'PostgREST 12 is out and we take a look at some of the major new features like JWT Caching and Aggregate Functions'
```
